src/components/general/SubActions.tsx

```diff
@@ -6,15 +6,19 @@ import WatchActionCard from './WatchActionCard';
 import { SubActionCardFragment } from 'common/__generated__/graphql';
 import { useTranslation } from 'common/i18n';
 
+const SubactionsHeader = styled.h2`
+  font-size: ${({ theme }) => theme.fontSizeLg};
+`;
+
 const ActionTabs = styled.div`
   display: flex;
   max-width: 100%;
   overflow-x: auto;
 `;
 
 const ActionTab = styled.button<{
-  isActive: boolean;
-  isEnabled: boolean;
+  $isactive: boolean;
+  $isenabled: boolean;
```

Comment:
Nitpick: camel casing

Reply:
True! I only just discovered the intricacies of styled component props. I think the console suggestion was either $ or lose camelCase and I opted for both!